app/models/notification_subscription.rb

```diff
@@ -10,11 +10,13 @@ class NotificationSubscription < ApplicationRecord
   belongs_to :user
 
   validates :config, presence: true, inclusion: { in: %w[all_comments top_level_comments only_author_comments] }
-  validates :notifiable_id, presence: true
```

Comment:
right - belongs to notifiable... I kind of like this "don't be redundant, belongs_to implies id present" rule. It does trip whenever you do something else, but it's free dead code removal!